node/pkg/driver/node_utils.go

```diff
@@ -239,6 +243,25 @@ func (n NodeUtils) ParseFCPorts() ([]string, error) {
 	return fcPorts, nil
 }
 
+func (n NodeUtils) IsFCExists() bool {
+	if n.IsPathExists(FCPath) {
+		return n.IsDirHasContent(FCPath)
+	}
+	return false
+}
+
+func (n NodeUtils) IsDirHasContent(path string) bool {
```

Comment:
this should be a private function (start with lower case letter).
also consider renaming to something like `isNotEmptyDir`, or even `isEmptyDir` with `!` before the call - @zingero what do you think?

Reply:
`isEmptyDir` is preferred. Personally, I don't like adding "not" as part of the function name since it can create double negatives.

Reply:
I used `IsDirHasContent` because I didn't want to use double negatives with `isNotEmptyDir` and with `return ! n.isEmptyDir(FCPath)`, so if you think that this `isEmptyDir` with the `!` is better so I will change it.
Anyway I will change the first character to a lower case letter.

Reply:
I like `!n.isEmptyDir(FCPath)` a bit better than `n.IsDirHasContent(FCPath)`, but whatever you guys prefer